src/Common/tests/Common.Tests.builds

```diff
@@ -3,9 +3,11 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="Common.Tests.csproj">
+      <TestTFMs>netcoreapp1.0;net46;net461;net462;net463</TestTFMs>
```

Comment:
>;net46;net461;ne [](start = 29, length = 16)

We should specify the minimum value of net46 only. I mean net46. the filtering should handle that net46 should work for net461, net462. in short, 

<TestTFMs>netcoreapp1.0;net46;net461;net462;net463</TestTFMs>

should be 

<TestTFMs>netcoreapp1.0;net46</TestTFMs>

by the way, I supported up to net462 in the filtering so we may need to add net463 too to the filtering